src/azure-cli/azure/cli/command_modules/iot/tests/latest/test_iot_commands.py

```diff
@@ -21,6 +21,16 @@ def test_iot_hub(self, resource_group, resource_group_location):
         ehConnectionString = self._get_eventhub_connectionstring(rg)
         subscription_id = self._get_current_subscription()
 
+        # Test hub life cycle in free tier
+        self.cmd('iot hub create -n {0} -g {1} --sku F1'.format(hub, rg), expect_failure=True)
+        self.cmd('iot hub create -n {0} -g {1} --sku F1 --partition-count 4'.format(hub, rg), expect_failure=True)
+        self.cmd('iot hub create -n {0} -g {1} --sku F1 --partition-count 2'.format(hub, rg),
+                 checks=[self.check('resourcegroup', rg),
+                         self.check('name', hub),
+                         self.check('sku.name', 'F1'),
+                         self.check('properties.eventHubEndpoints.events.partitionCount', '2')])
+        self.cmd('iot hub delete -n {0}'.format(hub), checks=self.is_empty())
+
```

Comment:
Could you please add the tests in the scenario where S1 is the default value of sku?

Reply:
Updated the test to use the default value of SKU in hub create command